meta-balena-common/recipes-connectivity/networkmanager/networkmanager_1.20.2.bb

```diff
@@ -19,7 +19,7 @@ DEPENDS = " \
     curl \
```

Comment:
nitpick, if you are going to push again, there is a typo in the commit log s/depricated/deprecated/

Reply:
will fix, thanks!